flake.nix

```diff
@@ -29,9 +34,21 @@
         pkgs = pkgsFor.${system};
       in {
         default = pkgs.mkShell {
-          inputsFrom = [
-            packages.${system}.default
+          buildInputs = with pkgs; [
```

Comment:
I still think `inputsFrom` should be used unless not possible.

Reply:
This was largely coming from the lengthy build time for llvm, I think, which isn't necessary for a development environment (for the most part).